OrbitCore/LinuxTracingHandler.cpp

```diff
@@ -93,11 +95,14 @@ void LinuxTracingHandler::OnCallstack(
   CallStack cs;
   cs.m_ThreadId = callstack.GetTid();
 
+  uint64_t unknownOffset = LinuxTracing::CallstackFrame::kUnknownFunctionOffset;
   for (const auto& frame : callstack.GetFrames()) {
     uint64_t address = frame.GetPc();
     cs.m_Data.push_back(address);
 
-    {
+    // TODO(kuebler): This is mainly for clustering IPs to their functions.
+    //  We should enable this also as a post-processing step.
+    if (frame.GetFunctionOffset() != unknownOffset) {
```

Comment:
I think that you should use `LinuxTracing::CallstackFrame::kUnknownFunctionOffset` directly here instead of creating the `unknownOffset` alias for it that hides the fact that this is a constant.

Reply:
I did it that way for readability, but I am fine to use the constant directly.